### PR TITLE
Update microsoft-defender-endpoint-android.md

### PR DESCRIPTION
**Pull request by shthota77**
Defender support for MAM scenario

### CODE REVIEW

**Review by shthota77 — COMMENTED** [review]
MAM Changes

**Comment by shthota77** [comment]
MAM Changes

**Comment by yogkumgit** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks

File: microsoft-365/security/defender-endpoint/microsoft-defender-endpoint-android.md [hunk 1]
```diff
@@ -63,7 +63,9 @@ This topic describes how to install, configure, update, and use Defender for End
     -   Access to the Microsoft 365 Defender portal.
 
         > [!NOTE]
-        > Microsoft Intune is the only supported Mobile Device Management (MDM) solution for deploying Microsoft Defender for Endpoint on Android. Currently only enrolled devices are supported for enforcing Defender for Endpoint on Android related device compliance policies in Intune. 
+        > Microsoft Defender for Endpoint can now extends the protection to organization’s data within a managed application for those who aren’t using an MDM but are using Intune to manage mobile applications. It also extends this support to customers who use other enterprise mobility management solutions, while still using Intune for [Mobile Application Management (MAM).](https://docs.microsoft.com/en-us/mem/intune/apps/mam-faq)
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/microsoft-defender-endpoint-android.md]:
```suggestion
        > Microsoft Defender for Endpoint can now extend the protection to an organization’s data within a managed application for those who aren’t using mobile device management (MDM) but are using Intune to manage mobile applications. It also extends this support to customers who use other enterprise mobility management solutions, while still using Intune for [mobile application management (MAM)](/mem/intune/apps/mam-faq).
```

File: microsoft-365/security/defender-endpoint/microsoft-defender-endpoint-android.md [hunk 2]
```diff
@@ -63,7 +63,9 @@ This topic describes how to install, configure, update, and use Defender for End
...
+>Prior to this update, Microsoft Defender for Endpoint worked on devices that were enrolled using Intune mobile device management (MDM) only.
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/microsoft-defender-endpoint-android.md]:
```suggestion
         > Prior to this update, Microsoft Defender for Endpoint worked on devices that were enrolled using Intune MDM only.  
```

File: microsoft-365/security/defender-endpoint/microsoft-defender-endpoint-android.md [hunk 3]
```diff
@@ -103,4 +105,5 @@ Guidance on how to configure Microsoft Defender for Endpoint on Android features
 ## Related topics
 - [Deploy Microsoft Defender for Endpoint on Android with Microsoft Intune](android-intune.md)
 - [Configure Microsoft Defender for Endpoint on Android features](android-configure.md)
+- >[What is mobile app management in Microsoft Intune? | How to get started with MAM.](https://docs.microsoft.com/en-us/mem/intune/apps/app-management#mobile-application-management-mam-basics)
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/microsoft-defender-endpoint-android.md]:
```suggestion
- [Mobile Application Management (MAM) basics](/mem/intune/apps/app-management#mobile-application-management-mam-basics)
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @shthota77 - kindly commit the suggested changes. Many thanks.

**Pull request merged**